src/lib/command/util/apps-util-user-input.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: rename to apps-user-input-create.ts to match apps-user-input-update.ts
```

Comment:
I plan to do this in a subsequent PR. I felt this one was getting too large already.